cardano-cli/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog for cardano-cli
 
+## 10.10.0.0
+
+- Bump consensus and network dependencies
+  (compatible, maintenance)
+  [PR 1189](https://github.com/IntersectMBO/cardano-cli/pull/1189)
+
+- Incorporate CIP-129
+  (feature, compatible)
+  [PR 1087](https://github.com/IntersectMBO/cardano-cli/pull/1087)
+
+- Simplify code by not using `QueryStakeAddressInfoCmdArgs` unnecessarily.
+  Also fixes `query spo-stake-distribution` command so that the output format flags are respected.
+  (refactoring, bugfix)
+  [PR 1184](https://github.com/IntersectMBO/cardano-cli/pull/1184)
+
+- `RIO` propagation 2025-05-07
+  (breaking, refactoring)
```

Comment:
Should this be included in the changelog? It does not say much. I don't think that was a PR with breaking changes too.